fineract-provider/src/main/resources/static/api-docs/velocity-doc.md

```diff
@@ -996,3 +996,256 @@ Response
 ```
 > This api returns all the vouchers for particular client which are going to be expired in next specified days.
 > For Example, fineract-provider/api/v1/vouchers/104/expiring/8 will returns all the vouchers for the client which are going to be expired in next 8 days
+
+### 13.  GET coupons/vouchers
```

Comment:
please confirm whether there is any filtering for this

Reply:
There is sorting there. 
for example,
fineract-provider/api/v1/datatables/Voucher/99?order=expiry asc will get the voucher in ascending order of expiry